operator/controllers/flowtest_controller.go

```diff
@@ -83,3 +120,60 @@ func (r *FlowTestReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&loggingplumberv1alpha1.FlowTest{}).
 		Complete(r)
 }
+
+func (r *FlowTestReconciler) checkForPassingFlowTest(ctx context.Context) error {
+	logger := log.FromContext(ctx)
+	flowTest := ctx.Value("flowTest").(loggingplumberv1alpha1.FlowTest)
+
+	var flows flowv1beta1.FlowList
+	if err := r.List(ctx, &flows, &client.MatchingLabels{"loggingplumber.isala.me/flowtest": flowTest.ObjectMeta.Name}); client.IgnoreNotFound(err) != nil {
```

Comment:
we should get the matching labels from a function

Reply:
This will get addressed in https://github.com/MrSupiri/rancher-logging-explorer/issues/16